hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -183,6 +186,47 @@ removed on `S3AFileSystem` level.
 </property>
 ```
 
+#### Fail on Error
+
+By default, S3AFileSystem write operations will still succeed when updates to
```

Comment:
Just a note that the recommendation to replace fs.s3a.metadatastore.fail.on.write.error with just failing on write error when auth mode is on has implications for this section. E.g. "By default" -> "In authoritative mode", etc.

Reply:
I'll revisit this pending a reply to https://github.com/apache/hadoop/pull/666#discussion_r276058353